aten/src/ATen/native/cuda/layer_norm_kernel.cu

```diff
@@ -722,6 +724,305 @@ void LayerNormKernelImpl(
       });
 }
 
+template<typename T, typename T_ACC> __device__
```

Comment:
guard the kernels behind macro so that they are not compiled on non-rocm platforms